README.md

```diff
@@ -63,19 +63,13 @@ See [docs/Readme.md](docs/Readme.md).
 
 ### Run DSP-API
 
-Create a test repository and load some test data into the triplestore:
+Create a test repository, load some test data into the triplestore, and start DSP-API:
 
 ```shell
-make init-db-test
+just stack-init-test
```

Comment:
As just is not a well known tool you might want to add a link to `https://just.systems/man/en/` and or add `brew install just`